### DISCUSSION TITLE
I can't give gift to Monika

### DISCUSSION THREAD

**Discussion by MonykaMeavelle** [Monika-After-Story/MonikaModDev#8533] [QA - tech support + general]
I can't give Monika gift. Im doing everything great, and she **can't read the file**
**can someone help me?**

**Comment by quinnyruu** [comment]
Did you close MAS? When I give her gifts, I close MAS first then do the stuff needed and wait for 30+ seconds

**Comment by ThePotatoGuy** [comment]
If they are spritepack-based gifts, then see [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation)

Otherwise, see [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#monika-doesnt-recognize-gifts).
